thirdparty/kyaml/runfn/runfn.go

```diff
@@ -110,6 +108,21 @@ func (r RunFns) Execute() error {
 	return r.runFunctions(nodes, output, fltrs)
 }
 
+func (r RunFns) getPipelineConfigFilterFn() (kio.LocalPackageSkipFileFunc, error) {
```

Comment:
Let's rename this method to `functionConfigFilterFunc` ?

Reply:
done